lib/workers/repository/process/lookup/index.ts

```diff
@@ -427,3 +429,23 @@ export async function lookupUpdates(
   }
   return res;
 }
+
+export function mergeConfigConstraints(
+  config: LookupUpdateConfig
+): LookupUpdateConfig {
+  if (config?.extractedConstraints) {
+    if (config?.constraints) {
+      for (const [constraint, value] of Object.entries(
+        config.extractedConstraints
+      )) {
+        if (is.nullOrUndefined(config.constraints[constraint])) {
+          config.constraints[constraint] = value;
+        }
+      }
+    } else {
+      config.constraints = config.extractedConstraints;
+    }
+    delete config.extractedConstraints;
+  }
+  return config;
+}
```

Comment:
i think you can achieve the same using this

```
const result = {
  ...config.extractedConstraints,
  ...config.constraints,
};
```

if they both have the same key, the second object will override the first object